scripts/dev/release_image.py

```diff
@@ -0,0 +1,65 @@
+import dockerutil
+import json
+import sys
+import argparse
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--old_repo_url", help="Url of the image to retag", type=str)
+    parser.add_argument(
+        "--new_repo_url", help="Url to where the new image should be pushed", type=str
+    )
+    parser.add_argument(
+        "--path",
+        help="path to use for the temporarily generated Dockerfile",
+        type=str,
+        default=".",
+    )
+    parser.add_argument("--release_file", help="Path to the release file", type=str)
+    parser.add_argument(
+        "--old_tag", help="the old tag of the image to retag", type=str,
+    )
+    parser.add_argument(
+        "--username", help="username for the registry", type=str,
+    )
+    parser.add_argument(
+        "--password", help="password for the registry", type=str,
+    )
+    parser.add_argument(
+        "--registry", help="The docker registry", type=str,
+    )
+    parser.add_argument(
+        "--labels", help="Labels for the new image", type=json.loads,
```

Comment:
how will `type=json.loads` work here? won't the type of `json.loads` be a function? Do we want this to be a `Dict` instead?

Reply:
That doesn't seem possible directly. All the accepted solutions are workarounds. I used this one: https://stackoverflow.com/questions/18608812/accepting-a-dictionary-as-an-argument-with-argparse-and-python but open to suggestions!

Reply:
Addendum from that topic:

```
The reason this works is actually quite interesting because internally argparse will just use the parameter value as a function to convert the argument. i.e. if type=int then it will use int(arg) or if type=json.loads then json.loads(arg)
```

Reply:
ah very cool!